### PR TITLE
Add worker manager to Mono Repo

### PR DESCRIPTION
**Pull request by jhford**
This PR is to add the worker manager codebase to the TC monorepo.  I  have followed the repoifier steps as best as I could, but I'm not 100% sure if I did it correctly.  I also need to see if the unit tests are working correctly with the monorepo, so please expect that this branch has failing unit tests until I can fix them.

### CODE REVIEW

**Comment by jhford** [comment]
I am removing the automatic review request because I'm not sure if the tests will pass or fail.

**Comment by imbstack** [comment]
It looks like it just needs added to the root `package.json`. It can either be added manually or with the `-W` flag that yarn requires to do things at that level. `yarn add --dev -W memory-streams` should work!

**Comment by imbstack** [comment]
This is an exciting PR. Thanks for this

File: services/worker-manager/schemas/v1/ruleset.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+$schema: http://json-schema.org/draft-06/schema#
```
Comment (djmitche) [inline services/worker-manager/schemas/v1/ruleset.yml]: This will need to point to the metaschema (see other `schemas/v1/*.yml` for an example)

**Comment by djmitche** [comment]
#124 should fix the permissions thing.

**Comment by djmitche** [comment]
I rebased and addressed the review comments.  I did a `taskcluster-builder build` and (as designed) the worker-manager service was automatically included in the resulting monoimage and started up just fine, happily serving me an empty array for `/api/worker-manager/v1/worker-configurations`.

**Comment by djmitche** [comment]
(We can build k8s deployments for this in a subsequent PR)

**Comment by djmitche** [comment]
@jhford I can't flag you for review but -- please have a look.  I'll get this landed tomorrow.

**Comment by djmitche** [comment]
Woo!

**Pull request merged**